Bio/SearchIO/HmmerIO/hmmer2_text.py

```diff
@@ -330,7 +330,7 @@ def __iter__(self):
         handle = self._handle
         handle.seek(0)
         start_offset = handle.tell()
-        regex_id = re.compile(b"Query\s*(?:sequence|HMM)?:\s*(.*)")
+        regex_id = re.compile(as_bytes(r"Query\s*(?:sequence|HMM)?:\s*(.*)"))
```

Comment:
This should probably have the double prefix of b (for bytes) and r (for raw, meaning leave the slashes as slashes):

```
>>> br"Here\tand\nthere"
b'Here\\tand\\nthere'
```